### PR TITLE
- Added wxwidgets version 3 support

### PR DESCRIPTION
**Pull request by alexhrv**

### CODE REVIEW

File: tce/configure.ac [hunk 1]
```diff
@@ -612,14 +612,14 @@ GUI_INCLUDES="$GUI_INCLUDES $($wxwindowsBin --cxxflags)"
 # if wx-config is found and the version is 2.4.*, conditional
 # WX is set.
 case "$($wxwindowsBin --version)" in
-     2.6.* | 2.8.* | 2.9.* )
+     2.6.* | 2.8.* | 2.9.* | 3.0.*)
```
Comment (pjaaskel) [inline tce/configure.ac]: Did you try if 3.1 works too?

File: tce/src/applibs/wxToolkit/MachineWindow/MachineCanvas.cc [hunk 2]
```diff
@@ -523,6 +523,7 @@ bool
 MachineCanvas::saveEPS(const std::string& filename, const std::string& title,
     const std::string& creator) {
 
+#if !wxCHECK_VERSION(3, 0, 0)
```
Comment (pjaaskel) [inline tce/src/applibs/wxToolkit/MachineWindow/MachineCanvas.cc]: Better comment out the whole method and also remove the possible menu items related to this until it works.

File: tce/src/procgen/ProDe/OperationDialog.cc [hunk 3]
```diff
@@ -1094,13 +1094,13 @@ OperationDialog::createContents(
 
     wxFlexGridSizer *item0 = new wxFlexGridSizer( 1, 0, 0 );
     item0->AddGrowableCol( 0 );
-    item0->AddGrowableCol( 1 );
+    //tem0->AddGrowableCol( 1 );
```
Comment (pjaaskel) [inline tce/src/procgen/ProDe/OperationDialog.cc]: Do not commit commented out code.

**Comment by linehill** [comment]
I get compiler warning about:
MainFrame.cc:744:36: warning: unused parameter 'event' [-Wunused-parameter]

**Pull request merged**